src/rpc/blockchain.cpp

```diff
@@ -926,6 +931,7 @@ static RPCHelpMan getblock()
                         {RPCResult::Type::OBJ, "", "",
                         {
                             {RPCResult::Type::ELISION, "", "The transactions in the format of the getrawtransaction RPC. Different from verbosity = 1 \"tx\" result"},
+                            {RPCResult::Type::NUM, "fee", "The transaction fee in " + CURRENCY_UNIT + ", omitted if efficient calculation isn't possible (due to pruning)"},
```

Comment:
Hm, I think since with the currently available version of pruning the block wouldn't be found anyway this would be slightly better and also more precise:
```suggestion
                            {RPCResult::Type::NUM, "fee", "The transaction fee in " + CURRENCY_UNIT + ", omitted if efficient calculation isn't possible (due to block undo data being unavailable)"},
```